src/Products/TemporaryFolder/LowConflictConnection.py

```diff
@@ -16,30 +16,33 @@
 from cPickle import Unpickler
 from cStringIO import StringIO
```

Comment:
cPickle and cStringIO don't exist in Python 3, right? So it's not clear to me how this could work there. Is there logic somewhere that conditionally imports this module (I don't see that in the PR)?

Reply:
Ah. That’s another module which isn’t even imported in any of the tests. As I said test coverage is virtually nonexistent for this project.

Reply:
Turns out that entire module is just old code and has been dysfunctional since ZODB 3.4 came out. There's neither a `_invalid` method on the connection nor is there `_version` support anymore.